README.md

```diff
@@ -12,13 +12,15 @@ Node.js plugin for [asdf](https://github.com/asdf-vm/asdf) version manager
 ### MacOS
 * [GNU Core Utils](http://www.gnu.org/software/coreutils/coreutils.html) - `brew install coreutils`
 * [GnuPG](http://www.gnupg.org) - `brew install gpg`
+* awk - any posix compliant implementation (tested on gawk `brew install gawk`)
```

Comment:
since asdf doesnt list awk as a depency. and it seems awk is already installed,at least on my mac is on /usr/local/bin, and, correct me if i am wrong, but brew does not use this directory right? i would just remove this line

Reply:
I never tested on MacOS, so I assumed that the default installed awk is not gawk, I put the brew installation command just for completeness. But i can remove it if it's better without it